scandium-core/src/main/java/org/eclipse/californium/scandium/dtls/EcdhPskServerKeyExchange.java

```diff
@@ -97,23 +110,50 @@
 	 * @throws NullPointerException if any of arguments ecdhe, 
 	 * clientRandom, serverRandom are {@code null}
 	 */
-	public EcdhPskServerKeyExchange(ECDHECryptography ecdhe, Random clientRandom, Random serverRandom, 
+	public EcdhPskServerKeyExchange(String hint, ECDHECryptography ecdhe, Random clientRandom, Random serverRandom, 
 			int namedCurveId, InetSocketAddress peerAddress) throws GeneralSecurityException {	
-		this(namedCurveId, peerAddress);
+		super(peerAddress);	
+		if (hint == null) {
```

Comment:
For convenience  we should also accept `null` here, don't you think? BTW you have not included the *hint* param in the list of params that would cause a NullPointerException anyway ;-)

Reply:
yes you are correct !, for convenience null is better, thank you for the suggestion